test/integration/discordactions.test.js

```diff
@@ -304,6 +326,26 @@ describe("Discord actions", function () {
         });
     });
 
+    it("should create reson and pass it down to the bot", async function () {
```

Comment:
create or delete?

Reply:
It is 'create'. An audit log for for delete operation

Reply:
But here it's delete right?
![image](https://github.com/Real-Dev-Squad/website-backend/assets/85296770/cda57d31-65c3-4efc-a85b-a284b2f2e1d4)

Reply:
I have changed the test name, now it should make sense. basically the an audit log is being created for deletion of the group